src/Development/IDE/Core/Compile.hs

```diff
@@ -479,7 +534,7 @@ loadModuleHome
     -> HscEnv
     -> HscEnv
 loadModuleHome mod_info e =
-    e { hsc_HPT = addToHpt (hsc_HPT e) mod_name mod_info }
+    e { hsc_HPT = addToHpt (hsc_HPT e) mod_name mod_info, hsc_type_env_var = Nothing }
```

Comment:
what's this change for?

Reply:
While debugging, I was just making sure everything was consistent with the way `GHC` was doing things: https://hackage.haskell.org/package/ghc-8.10.1/docs/src/GhcMake.html#local-6989586621681025499